onnxruntime/test/perftest/command_args_parser.cc

```diff
@@ -51,12 +51,18 @@ namespace perftest {
       "\t-d [cudnn_conv_algorithm]: Specify CUDNN convolution algothrithms: 0(benchmark), 1(heuristic), 2(default). \n"
       "\t-q: [CUDA only] use separate stream for copy. \n"
       "\t-z: Set denormal as zero. When turning on this option reduces latency dramatically, a model may have denormals.\n"
+      "\t-i: [OpenVINO only] Specify OpenVINO runtime options as key value pairs. Different options available are: \n"
+      "\t    [device_type]: Overrides the accelerator hardware type and precision with these values at runtime.\n"
```

Comment:
I think George wanted to avoid this list in the help command, as it will crowd the help for onnxruntime_perf_test if we add more options. Can we just have the first line and the example. Instead of describing the options. Maybe we can put a statement to look at BUILD.md for various valid options. What do you guys think ?

Reply:
This help message will be displayed after all building is done and when the user is trying to run the binary. It wouldn't be good UX to have them go back to the sources from this stage to find the Build.md for help. I think George was only concerned with adding options multiple command line arguments and wanted to them to instead be a single list of key-values pairs. I think we should display this information in the help message. We might need to just display it as a separate (maybe tabbed) segment so that these options don't seem mixed with the other perf test options.

Reply:
yep making it tabbed and indented under OpenVINO only option makes sense. It might be ok

Reply:
![image](https://user-images.githubusercontent.com/29036417/107613759-f3f42c00-6c6e-11eb-935d-59e97aee77f9.png)

I have put tabbed space for specifying the usage of various run-time options. I think this should give a clear picture to the end users.